liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -726,6 +728,27 @@ private void configureLogging(Level logLevel, String logFile, boolean wasDefault
         }
     }
 
+    /**
+     * Configures the system output stream and error stream based on supplied global parameter.
+     */
+    private void setOutputStream(Logger logger, LogService logService) {
+        final LogOutputStream logOutputStream = LiquibaseCommandLineConfiguration.LOG_STREAM.getCurrentValue();
+        final Level level = LiquibaseCommandLineConfiguration.LOG_LEVEL.getCurrentValue();
+        if (logOutputStream != null) {
+            Arrays.stream(logger.getHandlers()).forEach(logger::removeHandler); // Unsure of impact of this statement.
+            StreamHandler streamHandler = new StreamHandler(logOutputStream.getOutputStream(), new SimpleFormatter());
```

Comment:
What if one of the handlers you've removed on line 738 isn't a stream handler? You never put it back. Is that okay?

Reply:
Good point, looks like it works when I only remove handlers that are streamhandlers so I updated it to work that way. I'm unsure of the impact of removing all handlers.